src/routes/api/articles/commentController.js

```diff
@@ -0,0 +1,136 @@
+import models from '../../../models';
+// import db from '../../../models/index';
+
+const {
+  users, comment
+} = models;
+
+// const { articles, users, comment } = models;
+/**
+ *
+ *
+ * @export
+ * @class Articles
+ */
+class Comment {
+/**
+ *
+ *
+ * @static
+ * @param {*} req
+ * @param {*} res
+ * @param {*} next
+ * @memberof comment
+ * @returns {void}
+ */
+  static async postComment(req, res, next) {
+    const { body } = req.body;
+    let articleId, parentCommentId;
+    if (req.params.articlesId === undefined) {
+      parentCommentId = req.params.parentCommentsId;
+      articleId = null;
+    } else {
+      articleId = req.params.articlesId;
+      parentCommentId = null;
+    }
+    try {
+      const newComment = await comment.create({
+        articleId,
+        parentCommentId,
+        body,
+        author: req.user.id
+      });
+      if (newComment) {
+        return res.status(201).json({
+          comment: {
+            id: newComment.id,
+            body: newComment.body
+          },
+        });
+      }
+    } catch (err) {
+      if (err.parent.code === '23503') {
```

Comment:
Please do a check for an alternative implementation approach not tied to Postgres.

Reply:
Ok